### PR TITLE
modules: add interval files for picard collecthsmetrics

### PR DESCRIPTION
**Pull request by projectoriented**
Adds target and bait interval files for picard collecthsmetrics https://github.com/nf-core/modules/issues/793

### CODE REVIEW

**Comment by projectoriented** [comment]
Woops! Please don't merge yet if it looks good, I need to make sure these files work the module 😓

File: data/genomics/sarscov2/genome/baits.interval_list [hunk 1]
```diff
@@ -0,0 +1,7 @@
+@HD	VN:1.6	SO:coordinate
+@SQ	SN:MT192765.1	LN:29829	M5:c95f3e5592d0ad9974e41e7f0ea14eb0	UR:file:/Users/kevin/Documents/nfcore/modules/tests/data/fasta/sarscov2/GCA_011545545.1_ASM1154554v1_genomic.fna
```
Comment (drpatelh) [inline data/genomics/sarscov2/genome/baits.interval_list:2]: Does the UR need to be changed here to a raw github URL?

File: data/genomics/sarscov2/genome/baits.interval_list [hunk 2]
```diff
@@ -0,0 +1,7 @@
+@HD	VN:1.6	SO:coordinate
```
Comment (drpatelh) [inline data/genomics/sarscov2/genome/baits.interval_list:1]: I wonder whether we should put these files in a `picard` folder instead?

Comment (projectoriented) [reply data/genomics/sarscov2/genome/baits.interval_list]: done!

File: data/genomics/sarscov2/genome/baits.interval_list (hunk 1 above)
Comment (projectoriented) [reply data/genomics/sarscov2/genome/baits.interval_list]: done! :)

**Pull request merged**